daffodil-runtime1/src/main/scala/org/apache/daffodil/dpath/DPathRuntime.scala

```diff
@@ -311,3 +311,18 @@ abstract class Converter extends RecipeOp {
 trait ToString extends Converter {
   override def computeValue(a: DataValuePrimitive, dstate: DState): DataValueString = a.getAnyRef.toString
 }
+
+
+/**
+ * In some cases, expressions that are never used lack context and so cannot be compiled.
+ * Because they aren't used, it it safe to ignore them, but parts of Daffodil code still
+ * require a CompiledExpression. In these cases, we can use this RuntimeAbort recipe in
```

Comment:
Can you add description of an example of something that requires a compiled expression even though unused?

Reply:
Done